### PR TITLE
doc: release notes for December release

### LINKED ISSUE #1218: doc: release notes for December release

### PR DESCRIPTION
**Pull request by jsilvela**
Closes #1218 

Signed-off-by: Jaime Silvela <jaime.silvela@enterprisedb.com>

### CODE REVIEW

**Review by jlong49 — APPROVED** [review]

File: docs/src/release_notes/v1.16.md [hunk 1]
```diff
@@ -6,6 +6,55 @@ For a complete list of changes, please refer to the
...
+
+- Recognizing Armando Ruocco (@armru) as a new CloudNativePG maintainer for his
+  consistent and impactful contributions (#1167)
+- FINAL patch release for 1.16: 1.16.5. Release 1.16 reaches end of life.
+
+Enhancements:
+
+- Add rpm/deb package for kubectl-cnpg plugin (#1008)
+- Update default PostgreSQL version for new cluster definitions to 15.1 (#908)
+- Remove ARMv7 support (#1092)
```
Comment (mnencia) [inline docs/src/release_notes/v1.16.md:27]: I do not see this fitting the Enhancements section. I would move it in the announcements

File: docs/src/release_notes/v1.17.md [hunk 2]
```diff
@@ -6,6 +6,56 @@ For a complete list of changes, please refer to the
...
+  - Troubleshooting emergency backup instructions (#1184)
+
+Fixes:
+
+- Ensure PGDATA permissions on bootstrap are properly set to 750 (#1164)
+- Ensure the PVC containing WALs is deleted when scaling down the cluster
+(#1135)
+- Ensure that we create secrets and services only when not found (#1145)
+- Respect configured pg-wal when restoring (#1216)
+- Respect configured pg-wal when restoring (#1216)
```
Comment (mnencia) [inline docs/src/release_notes/v1.17.md:46]:
```suggestion
- Respect configured pg-wal when restoring (#1216)
```

File: docs/src/supported_releases.md [hunk 3]
```diff
@@ -100,6 +99,8 @@ by the CloudNativePG maintainers).
 | Version         | Release Date      | End of Life              | Compatible Kubernetes Versions |
 |-----------------|-------------------|--------------------------|--------------------------------|
 | 1.15.x          | April 21, 2022    | October 6, 2022          | 1.21, 1.22, 1.23               |
+| 1.16.x          | July 7, 2022      | December 20, 2022      | 1.22, 1.23, 1.24              |
```
Comment (mnencia) [inline docs/src/supported_releases.md:102]:
```suggestion
| 1.16.x          | July 7, 2022      | December 21, 2022      | 1.22, 1.23, 1.24              |
```

**Review by mnencia — APPROVED** [review]

File: docs/src/release_notes/v1.16.md (hunk 1 above)
Comment (jsilvela) [reply docs/src/release_notes/v1.16.md:27]: right you are, will move

**Pull request merged**